models/model.js

```diff
@@ -69,13 +69,31 @@ const fetchCommentsByArticleId= (article_id) =>{
   if (!article_id ||!username ||!body) {
     return Promise.reject({ status: 400, msg: 'Incomplete comment' });
   } 
-   
+
+  else{   
     return db.query(`INSERT INTO comments (article_id, author, body) VALUES ($1, $2, $3) RETURNING *;`, [article_id, username, body])
     .then((result) => { 
      
         return result.rows[0]    
   })
 }
+  }
+
+const fetchVotedArticlesById = (article_id, votes) => {
+if (typeof votes !== 'number') {
+    return Promise.reject({ status: 400, msg: 'Invalid article_id -- must be an integer' });
+  }   
+  return db.query(`UPDATE articles 
+  SET votes = votes + $2 
+  WHERE article_id = $1 RETURNING * ;`
+  , [article_id, votes])
```

Comment:
This is purely a preference but I would have $2 and $1 in order in the query for easier readability 📖